### DISCUSSION TITLE
Square player texture

### DISCUSSION THREAD

**Discussion by ghost** [MageFlight/game-dev-club#8] [Polls]
![squarePlayerTextureFailure](https://github.com/MageFlight/game-dev-club/assets/128601345/c8f5167d-0596-4970-add3-e0bf809c3c9f)

**Comment by MageFlight** [comment]
## My two cents
 - The color scheme could use more variety
 - None of the trapezoids are straight
 - What theme is this trying to fit in?
 - There should be something in the middle of the square - your eyes are drawn to the middle, but there is nothing there. Maybe there could be some sort of symbol that relates to the story?

**Reply by ghost** [reply]
Who makes the lore
